website/docs/fun/qotd/advanced/overview.md

```diff
@@ -0,0 +1,102 @@
+- Pings a role on each new question.
+- Sticky message at the bottom of the QOTD channel reminding members what the question is.
+- Completely interactive configuration setup, no config variables required.
+
+:::info
+
+Most of these features are configurable via the interactive setup!
+
+:::
+
+## Installing
+
+As usual, there are instructions describing where to put the script and which trigger to use on the pages corresponding to the individual commands. Additionally, we've documented how and where to add these scripts down below.
+
+### Main Command
+
+Add the [main command](main-cc) as a new custom command. The trigger is a regex type with the trigger set to `\A`.
+
+### Component Handler
+
+Next, add the [component handler](component-handler) as a new custom command. The trigger is a component type with the
+Custom ID set to `\Aqotd-`.
+
+### Modal Handler
+
+Next, add the [modal handler](modal-handler) as a new custom command. The trigger is a modal type with the
+Custom ID set to `\A0?qotd-`.
+
+### Interval Trigger
+
+Finally, add the [interval command](interval) as a new custom command. The trigger is an hourly interval type with the
+interval set to `24` (or whatever interval you'd like). Set the `channel` menu to any channel. Then, disable the command
+*before saving it*. This will prevent it from running before we've configured the QOTD system, which would cause it to error.
+
+## Configuration
+
+Now, it's time to configure your QOTD system. This is a completely interactive setup, so just run `-qotd setup` (or
+replace `-` with your server's prefix) and follow the instructions.
+
+:::danger
+
+YAGPDB must have `View Channel`, `Send Messages`, and `Embed Links` permission in the channel you run `qotd setup` in.
+
+:::
+
+:::info
+
+Once you are finished with setup, you may now enable your [interval cc](#interval-trigger). If you forget to do this,
+your questions will not automatically post.
+
+:::
+
+## Usage
+
+Commands:
+`qotd setup` - Starts the interactive setup. Run this at any time to reconfigure QOTD.
+`qotd suggest [new question]` - Adds a question to the suggestion queue (if Suggestions Open is enabled during setup)
+`qotd push` - Pushes a new question immediately
+`qotd delete [ID or -all]` - Deletes a question from the queue by ID or resets the queue to blank. ID is the question's
+place in the queue.
+
+Suggesting:
+`qotd setup` Command: Adds a new question to the queue. Can be run anywhere YAGPDB has view messages permission.
+Buttons: Buttons are located under each question, and under the question queue message. Tap them to open a modal to type
+your question into.
+Queue Channel: Sending any message in the queue channel will add a question to the queue.
+
+:::danger
+
+It is recommended that you either remove `Send Messages` permissions from your server members in your Queue channel, or
+set a substantial slowmode to prevent users from spamming questions.
```

Comment:
```suggestion
It is recommended that you either remove `Send Messages` permissions from your server members in your Queue channel, or
set an appropriate slowmode to prevent users from spamming questions.
```